deploy/docker/watchtower-hooks/pre-check.sh

```diff
@@ -0,0 +1 @@
+exit 0
```

Comment:
We are making these files _executable_, with the `chmod +x` in the `Dockerfile`. For scripts that'll be executable, it's best to always have a shebang line. It _may_ be okay for us even without it, since these scripts only ever run on Ubuntu, but I'd still advise including the shebang line.


```suggestion
#!/usr/bin/env bash
exit 0
```

Same in the `pre-update.sh` script as well.

Reply:
chore: review change